modules/configuring-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,134 @@
+
+:FeatureName: VMware vSphere region and zone enablement
+include::snippets/technology-preview.adoc[]
+
+The default `install-config.yaml` file configuration from the previous release of {product-title} is deprecated. You can continue to use the deprecated default configuration, but the `openshift-installer` will prompt you with a warning message that indicates the use of deprecated fields in the configuration file.
+
+[IMPORTANT]
+====
+The example uses the `govc` command. The `govc` command is an open source command available from VMware; it is not available from Red Hat. The Red Hat support team does not maintain the `govc` command. Instructions for downloading and installing `govc` are found on the VMware documentation website
+====
+
+.Prerequisites
+* You have an existing `install-config.yaml` installation configuration file.
++
+[IMPORTANT]
+====
+You must specify at least one failure domain for your {product-title} cluster, so that you can provision datacenter objects for your VMware vCenter server. Consider specifying multiple failure domains if you need to provision virtual machine nodes in different datacenters, clusters, datastores, and other components.
+====
+
+.Procedure
+
+. Enter the following `govc` command-line tool commands to create the `openshift-region` and `openshift-zone` vCenter tag categories:
++
+[IMPORTANT]
+====
+If you specify different names for the `openshift-region` and `openshift-zone` vCenter tag categories, the installation of the {product-title} cluster fails.
+====
++
+[source,terminal]
+----
+$ govc tags.category.create -d "OpenShift region" openshift-region
+----
++
+[source,terminal]
+----
+$ govc tags.category.create -d "OpenShift zone" openshift-zone
+----
+
+. To create a region tag for each region vSphere datacenter where you want to deploy your cluster, enter the following command in your terminal:
++
+[source,terminal]
+----
+$ govc tags.create -c <region_tag_category> <region_tag>
+----
+
+. To create a zone tag for each vSphere cluster where you want to deploy your cluster, enter the following command:
++
+[source,terminal]
+----
+$ govc tags.create -c <zone_tag_category> <zone_tag>
+----
+
+. Attach region tags to each vCenter datacenter object by entering the following command:
++
+[source,terminal]
+----
+$ govc tags.attach -c <region_tag_category> <region_tag_1> /<datacenter_1>
+----
+
+. Attach the zone tags to each vCenter datacenter object by entering the following command:
++
+[source,terminal]
+----
+$ govc tags.attach -c <zone_tag_category> <zone_tag_1> /<datacenter_1>/host/vcs-mdcnc-workload-1
+----
+
+. Change to the directory that contains the installation program and initialize the cluster deployment according to your chosen installation requirements.
+
+.Sample `install-config.yaml` file with multiple datacenters defined in a vSphere center
+
+[source,yaml]
```

Comment:
Hi @rvanderp3 . Joe confirmed that this configuration will not work in 4.12 (config was taken from 4.13 published docs). Best to copy and paste the example from your PR?